src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="picture imac"
+        class="card__picture"
+      >
+
+      <p class="card__product-name">
+        Моноблок APPLE A1419 iMac 27&#34;
+        Retina 5K (MNED2UA/A)
+      </p>
+
```

Comment:
я бы добавил родительский блок здесь и в других местах, так как в видео по bem говорилось, что нельзя задавать внешние отступы элементу напрямую - только родительскому блоку
 ```suggestion
  <div>
      <p class="card__product-name">
        Моноблок APPLE A1419 iMac 27&#34;
        Retina 5K (MNED2UA/A)
      </p>
  </div>

```